test/e2e/framework/util.go

```diff
@@ -339,7 +339,7 @@ func CreateTestingNS(ctx context.Context, baseName string, c clientset.Interface
 	// We don't use ObjectMeta.GenerateName feature, as in case of API call
 	// failure we don't know whether the namespace was created and what is its
 	// name.
-	name := fmt.Sprintf("%v-%v", baseName, RandomSuffix())
+	name := fmt.Sprintf("%v-%d-%v", baseName, ginkgo.GinkgoParallelProcess(), RandomSuffix())
```

Comment:
check line 365, it adds a random suffix just to avoid this ... have you seen this happening ?

Reply:
Thanks, I will re-check my test log to see if this happened.